<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output_configure</class>
 <widget class="QWidget" name="output_configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>output_configure</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="changeButton">
     <property name="text">
      <string>change</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="yesButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
